plugins/topology/src/components/Topology/TopologyComponent.tsx

```diff
@@ -34,13 +42,19 @@ export const TopologyComponent = () => {
     'replicasets',
   ];
 
-  const k8sResponseData = useAllWatchResources(watchedResources);
+  const k8sResourcesClusters = useK8sResourcesClusters(k8sObjectsResponse);
 
   return (
-    <K8sResourcesContext.Provider value={k8sResponseData}>
-      <div className="pf-ri__topology">
-        <TopologyWorkloadView />
-      </div>
-    </K8sResourcesContext.Provider>
+    <K8sResourcesClustersContext.Provider value={k8sResourcesClusters.clusters}>
+      <K8sResourcesClusterContext.Provider value={clusterContext}>
```

Comment:
We can have both the values in one context but since both of them gets set at different places so just to be more elaborate kept them in different contexts as well.